modules/time.js

```diff
@@ -0,0 +1,7 @@
+const time = () => {
+  const timeDate = document.getElementById('date');
+  const date = new Date();
+  timeDate.innerHTML = date;
+};
+
+export default time;
```

Comment:
- [ ] Even though you did a good job of ensuring that date and time are included in your project, It needs to conform to the project requirements. According to the PR, you are to use the `Luxon library` to get the current date and date. So i will suggest you implement that requirement to help you learn how to use a new tool effectively with little externally curated support. Exactly as you will do in your future job. :100: I have attached a screenshot of the project requirement below. 
- Project Requirement 
![image](https://user-images.githubusercontent.com/102699958/220166492-590488a7-bff4-4b74-8748-828a0170ca32.png)